sdk/core/System.ClientModel/src/System.ClientModel.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains building blocks for clients that call cloud services.</Description>
+    <Version>1.0.0-beta.1</Version>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
```

Comment:
Why disable analyzers?

Reply:
Why would we do it differently from Azure.Core?
https://github.com/Azure/azure-sdk-for-net/blob/feature/modelReaderWriter/sdk/core/Azure.Core/src/Azure.Core.csproj#L13

Reply:
Because Azure.Core is an azure library.